src/app/move-popup/sockets.html

```diff
@@ -1,13 +1,24 @@
 <div class="item-socket-category" ng-repeat="category in $ctrl.sockets.categories | orderBy:'category.index' track by category.category.hash">
-  <div class="item-socket-category-name">{{category.category.displayProperties.name}}</div>
+  <div class="item-socket-category-name">
+    <div>{{category.category.displayProperties.name}}</div>
+    <div ng-if="$ctrl.anyBestRatedUnselected(category)" class="best-rated-key">
```

Comment:
maybe add the `press-tip='vm.bestRatedText'` here as well? so if people hover they have a little more detail about what 'suggest perks' comes from?